SP5WWP/m17-coder/m17-coder-sym.c

```diff
@@ -6,8 +6,8 @@
 
 //libm17
 #include <m17.h>
-//tiny-AES-c
-#include "../../tiny-AES-c/aes.h"
+//tinier-aes
```

Comment:
Can we have .h and .c as separate files?

Reply:
Maybe, I'll look at it and see. We really only would need the prototype functions for the wrappers in an .h file, I think, the rest can still be referenced internally by the .c file. My intention for the way the aes.c file was written was that everything it needed was handled internally. In practice, I'll look at it and see how it goes, make some changes, and see if they all gel properly.